### ISSUE TITLE
the game cannot use the latest versions of JAWS

### ISSUE THREAD

**Issue by soundmud** [soundmud/soundrts#100]
ScreenReaderAPI.dll cannot use the latest versions of JAWS.
The license of the successor of ScreenReaderAPI isn't compatible with the BSD license.

**Comment by sanslash332** [comment]
@soundmud witch successor of screenReaderApi.dll are you trying to use?

I know two updated dlls. UniversalSpeetch.dll, and tolk.dll. any of them are useful?

**Comment by soundmud** [comment]
UniversalSpeech is the successor of ScreenReaderAPI. The license depends on the project license.
https://github.com/qtnc/UniversalSpeech

> Universal speech is distributed in several different licenses. [...] As long as your program is free software, you can use universal speech for free in your program, subject to the GPL license.

GPL is not LGPL, so if I use the library, I have to change the license of SoundRTS to GPL.

Tolk is LGPL. It might be a good replacement.
https://github.com/dkager/tolk

Another option could be accessible_output, but it could be tricky to package, or to install, and might be Python 2 only. I cannot find the homepage, just a pypi page from 2010.

I think that the interest of the dll is to ease the packaging and to have a library used by many languages.

**Comment by soundmud** [comment]
Tolk doesn't seem to detect SAPI if everything fails. Not a good sign.

**Comment by soundmud** [comment]
Maybe it would be simpler to start from kvark128 's Jaws code after all.
